src/main/kotlin/app/revanced/patches/music/misc/tastebuilder/TasteBuilderPatch.kt

```diff
@@ -12,7 +12,7 @@ import com.android.tools.smali.dexlib2.iface.instruction.TwoRegisterInstruction
 
 @Patch(
     name = "Hide taste builder",
-    description = "Hides the \"Tell us which artists you like\" card from homepage.",
+    description = "Hides the \"Tell us which artists you like\" card from the homepage.",
```

Comment:
I wasn't able to get this card to show on any of my accounts, unless it is the fullscreen popup where it asks you to select 5 artists. The fullscreen popup wasn't titled "Tell us which artists you like" so if this patch is referring to the fullscreen popup I experience the patch description should be changed.

Reply:
This patch is old enough that I'm not even sure if it's still valid.
One thing is for sure, these pop-ups will only appear on newly created Google accounts.

Reply:
It's difficult to properly test, but I got this on a new account in the unpatched app but never in the patched app.

![Screenshot_20240112_011232_YouTube Music.jpg](https://github.com/inotia00/revanced-patches/assets/84587632/1b3b72be-2724-4a63-9595-e6c1830db4b8)

Is this the popup the patch targets?

Reply:
Yeah.. maybe this is the patch that removes this screen.

This patch is from Vanced Music, so only @OxrxL, who was the maintainer of Vanced Music, would know exactly which screens this patch removes.

Reply:
> Yeah.. maybe this is the patch that removes this screen.
> 
> This patch is from Vanced Music, so only @OxrxL, who was the maintainer of Vanced Music, would know exactly which screens this patch removes.

There was a bottom panel on homepage, about the personal music tastes personalization on old YTM versions.

If layout is changed, is highly probably that this patch is became useless.